src/Sulu/Bundle/WebsocketBundle/.gitignore

```diff
@@ -0,0 +1,15 @@
+/Tests/app/cache/*
+/Tests/app/logs/*
+/node_modules/
+/vendor/
+/composer.phar
+/composer.lock
+/.sass-cache/
+*~
+
+# IDEs
+.idea/*
+*.iml
+
+# temp files
+TEMP_*.php
```

Comment:
If you don't encounter this files anytime (surfing the app/website, running the tests, ...) I would say yes.